drivers/watchdog/Kconfig.nrfx

```diff
@@ -11,15 +11,21 @@ config WDT_NRFX
 	help
 	  Enable support for nrfx WDT driver for nRF MCU series.
 
+# TODO: bouncing this information through Kconfig is not really necessary.
```

Comment:
Happy to drop this part of the patch if it's controversial.

Reply:
I'd prefer to drop this comment for now. It seems that these `NRFX_*` Kconfig options need more discussion. I'm not convinced they can be simply removed.